packages/shared/src/components/filters/AdvancedSettings.spec.tsx

```diff
@@ -137,7 +151,7 @@ it('should mutate update feed advanced settings', async () => {
       variables: { settings: params },
     },
     result: () => {
-      mutationCalled = true;
+      advancedSettingsMutation();
```

Comment:
Why did you change this mechanism?

Reply:
I initially was trying to contain the condition in a single jest mock to see if it will be called twice, but then I decided to separate them for clarity, then I just kept it to be consistent with each other.

Reply:
I find this new way less readable. Can you revert please everywhere?

Reply:
Sure thing.